spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index ' do
+  before :each do
```

Comment:
- [ ] As required for the project, kindly ensure that all your tests are passing as right now, all of them are failing.
![image](https://github.com/Wils-FOURR-speed/Blog-App-Project/assets/83042742/bd04de58-6c4e-482c-8c81-3a7e23233124)

Reply:
- [ ] You did a great job working on this milestone and implementing the changes requested by the previous reviewer! 💯 However, when I try to run your tests, they all fail. I noticed that you have a mismatch between your schema and your app code, for example, in your schema, for users, you have the property `post_counter` but in your app (User/model) you're using `posts_counter` and the same happens with the counters of `Post`. Please consider fixing all these typos and bugs! 🤓 🔧